waffle/management/commands/waffle_sample.py

```diff
@@ -51,7 +60,7 @@ def handle(self, *args, **options):
             try:
                 sample = Sample.objects.get(name=sample_name)
             except Sample.DoesNotExist:
-                raise CommandError('This sample does not exist.')
+                raise CommandError("This sample doesn't exist.")
```

Comment:
Single quotes. I find "does not" to be more understandable (esp. for non-native English speakers) than "doesn't".